src/globus_sdk/transport/requests.py

```diff
@@ -162,8 +163,9 @@ def tune(
         In particular, this can be used to temporarily adjust request-sending minutiae
         like the ``http_timeout`` used.
 
-        :param verify_ssl: Explicitly enable or disable SSL verification
-        :type verify_ssl: bool, optional
+        :param verify_ssl: Explicitly enable or disable SSL verification,
+            or configure the path to a CA certificate bundle to use for SSL verification
+        :type verify_ssl: bool, str, or pathlib.Path, optional
```

Comment:
This same doc update needs to be reflected in the `__init__` parameter docs in the class docstring.

Reply:
Oh! It's in the class docstring! It seemed weird to me that `.__init__` was completely undocumented. 😆

Reply:
New rev coming